test_huggingface_live.py

```
@@ -0,0 +1,313 @@
+    # Check sentence-transformers
+    try:
+        import sentence_transformers
+        print(f"✓ sentence-transformers installed (version: {sentence_transformers.__version__})")
+    except ImportError:
+        print("⚠ sentence-transformers not installed (optional for local embeddings)")
+        print("  Install with: pip install sentence-transformers")
+
+    # Check langchain packages
+    try:
+        import langchain_core
+        print(f"✓ langchain-core installed")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)